app/controllers/sign_up_controller.rb

```diff
@@ -36,6 +36,16 @@ def confirm_email
     end
   end
 
+  def update_email
```

Comment:
Why we are changing email in sing_up controller? It should be done in Users controller.

Reply:
And do not forget to update swagger.

Reply:
Fixed. Added swagger.